docker/Dockerfile.faux

```diff
@@ -23,8 +23,9 @@ FROM daq/aardvark:latest
 # Run this separately so it can be shared with other builds.
 RUN $AG update && $AG install openjdk-8-jre
 RUN $AG update && $AG install openjdk-8-jdk git
+RUN $AG update && $AG install curl
```

Comment:
Why add this again? (it's on the next line)

Reply:
fixed